pkg/cosign/tlog.go

```diff
@@ -57,6 +57,13 @@ type RekorPubKey struct {
 	Status tuf.StatusKind
 }
 
+// This is a map of RekorPubKeys indexed by log ID that's used in verification
```

Comment:
Love it. Another approach we could've taken is just to define a new type without an embedded Keys in it, something like:
type TrustedRekorPubKeys map[string]RekorPubKey

And that would've provided us with the documentation, but this is totes fine with me. By doing it this way in case there's something else that we may need in the future, we can add it easier here :)

Reply:
Yep, originally I used that approach of `type TrustedRekorPubKeys map[string]RekorPubKey`, but it ended up being ugly to do things like `AddKeyToTrustedRekorPubKeys` without supplying them as args rather than a method on a struct

Reply:
ah, perfect, yeah, I like this way personally better anyways :) Just jotted it down for prosperity (alternatives, considered lol)